package.json

```diff
@@ -111,6 +111,7 @@
     "libp2p-tcp": "^0.14.1",
     "libp2p-webrtc-star": "^0.18.0",
     "libp2p-websockets": "^0.13.1",
+    "mocha": "^8.0.1",
```

Comment:
What is the reason to require mocha here?

Reply:
seemed to be required to run the unit tests in intellij.  🤷

Reply:
This is strange. I do not have intellij to test this out, but running the tests in the CLI does not require `mocha` as we are not using it